mkosi.md

```diff
@@ -320,23 +320,13 @@ a boolean argument: either "1", "yes", or "true" to enable, or "0",
   Linux, additional repositories must be passed in the form `<name>::<url>`
   (e.g. `myrepo::https://myrepo.net`).
 
-`UseHostRepositories=`, `--use-host-repositories`
-
-: This option is only applicable for RPM-based distributions:
-  *CentOS*, *Fedora Linux*, *Mageia*, *Rocky Linux*, *Alma Linux*
-  and *OpenMandriva*.
-  Allows use of the host's existing RPM repositories.
-  By default, a hardcoded set of default RPM repositories is generated and used.
-  Use `--repositories=` to identify a custom set of repositories to be enabled
-  and used for the build.
-
-`RepositoryDirectory`, `--repository-directory`
+`RepositoryDirectories`, `--repository-directory`
```

Comment:
We already have a few cases, where the command line option is singular and the config file option is plural, but do we need to add more? Since this is a comma-delimited list that goes in as a single argument, I'd opt for
```suggestion
`RepositoryDirectories`, `--repository-directories`
```

Reply:
Yeah, but when on the CLI, you're able to use this option as `--repository-directory` with a single argument. I think for now it's nice to be consistent with the other options in this regard.